src/gennylib/src/PoolManager.cpp

```diff
@@ -79,3 +79,11 @@ mongocxx::pool::entry genny::PoolManager::client(const std::string& name,
     }
     return std::move(*entry);
 }
+
+std::unordered_map<std::string, size_t> genny::v1::PoolManager::instanceCount() {
+    auto out = std::unordered_map<std::string, size_t>();
```

Comment:
Add `std::lock_guard<std::mutex> getLock{this->_poolsGet}` above this line? I realize this is meant to only be used for testing purposes but it seems easy enough to make it thread-safe.

Reply:
Good idea, done

Reply:
You can use `std::lock_guard` instead of `std::unique_lock` because you aren't doing anything fancy with unlocking the mutex partway through. (This came up before in https://github.com/mongodb/genny/pull/24#discussion_r218973538.)